src/components/NearbyShops.vue

```diff
@@ -26,9 +19,9 @@ export default
             return this.$store.getters.nearbyShops;
         },
     },
-    methods:
+    components:
     {
-
+        ShopList,
```

Comment:
I am proposing to give ShopList more explicit name as we will use similar lists of shops in other components in the future and better to be sure what we exactly mean by the ShopList. What, Why or how???

Reply:
Right now we only have one shop list -> ShopList. If there arise a need for having multiple flavours of said list then I'll gladly rename it. Right now it serves a purpose of listing shops and we list shops in one way, so if  you don't suggest adding "View" to I see no point for name change ;)

Reply:
Please do it just to go with convention that We are not greedy with safe explaining names, ect... better to have this in code than in comment, as code is compiled and run and comment can mislead...This was very well explained by You and that should be fallowed as Must Be practice...
Accepted.